onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -277,8 +282,11 @@
~      // of fusion patterns and target on CPU. However, NCHWCtransformer will reorder the layout to nchwc which is only available for
~      // x86-64 cpu, not edge cpu like arm. But This tranformer could be used by opencl-ep/cpu-ep. So
       // we will prefer NhwcTransformer once ort runs on x86-64 CPU, otherwise ConvAddActivationFusion is enabled.
       // this PR #6351 implemented similiar fusion-pattern but only for CUDA, and can only fuse conv-add-relu, while we can fuse more activation.
       transformers.emplace_back(std::make_unique<ConvAddActivationFusion>(cpu_ep));
+
```

Comment:
>Dml::GraphTransformer [](http://example.com/codeflow?start=49&length=21)

Will this line need a `#ifdef USE_DML`? Try building your change locally without `--use_dml` too.


```
error: ld returned 1 exit status
undefined reference to `vtable for Dml::GraphTransformer'
```

[https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=692222&view=logs&j=7536d2cd-87d4-54fe-4891-bfbbf2741d83&t=32a004b5-6abc-5719-dbf1-16350e4ba258&l=2225](https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=692222&view=logs&j=7536d2cd-87d4-54fe-4891-bfbbf2741d83&t=32a004b5-6abc-5719-dbf1-16350e4ba258&l=2225)

Reply:
Actually, I had fixed this in my local and thought that I had pushed 3rd iteration. But certainly, I missed to push that fix before your review. I will push it along with acknowledging other comments.